components/dashboard/src/projects/ProjectListItem.tsx

```diff
@@ -40,7 +41,14 @@ export const ProjectListItem: FunctionComponent<ProjectListItemProps> = ({ proje
                                 menuEntries={[
                                     {
                                         title: "New Workspace",
-                                        href: `/#${project.cloneUrl}`,
+                                        href: gitpodHostUrl.withContext(`${project.cloneUrl}`).toString(),
+                                        separator: true,
+                                    },
```

Comment:
**thought**: From what I understand this will keep the existing new workspace menu entry with no options and was wondering it if we still need it, but this can become clearer later. Also, it doesn't hurt to keep this here for now, since it's hidden behind the more actions menu.

| BEFORE | AFTER |
|-|-|
| <img width="358" alt="dropdown-before" src="https://user-images.githubusercontent.com/120486/211617305-017f71a5-9c8b-4a50-a59c-4920bd989bb9.png"> | <img width="358" alt="dropdown-after" src="https://user-images.githubusercontent.com/120486/211617309-00cc521b-19a5-477f-a4d2-2472609d3374.png"> |